test/Business_Logic/MobiusDAO/README.md

```diff
@@ -0,0 +1,99 @@
+# MobiusDAO
+
+- **Type:** Exploit
+- **Network:** BSC
+- **Total lost:** ~2.15MM
+- **Category:** Bad Arithmetic
+- **Vulnerable contracts:**
+- - [Exploited Contract](https://bscscan.com/address/0x95e92b09b89cf31fa9f1eca4109a85f88eb08531) (Unverified Implementation)
+- **Tokens Lost**
+- - ~9.7 Quadrillion MBU
+
+- **Attack transactions:**
+- - [Attack Tx](https://bscscan.com/tx/0x2a65254b41b42f39331a0bcc9f893518d6b106e80d9a476b8ca3816325f4a150)
+
+- - Deployer EOA: [0xB32A53Af96F7735D47F4b76C525BD5Eb02B42600](https://bscscan.com/address/0xB32A53Af96F7735D47F4b76C525BD5Eb02B42600)
+
+- **Attack Block:**: 49470430
+- **Date:** May 11, 2025
+- **Reproduce:** `forge test --match-contract Exploit_MobiusDAO -vvv`
+
+## Step-by-step Overview
+
+1. Set up a malicious contract
+2. Call MobiusDAO's `deposit` function through the exploit contract with 0.001 WBNB
```

Comment:
This step-by-step is lacking to me, yes you can obviously execute via the exploit contract but the interesting part is how the exploit contract leverages the issue to allow the caller  to get 9.7 quadrillon tokens.